synapse/models/infotech.py

```diff
@@ -301,6 +301,16 @@ def getModelDefs(self):
                 ('it:app:snort:hit', ('guid', {}), {
                     'doc': 'An instance of a snort rule hit.',
                 }),
+                ('it:reveng:function', ('guid', {}), {
+                    'doc': 'A hash identifying a function',
+                }),
+                ('it:reveng:function:string', ('comp', {'fields': (('function', 'it:reveng:function'), ('string', 'str'))}), {
+                    'doc': 'A reference to a string inside a function',
+                }),
+                ('it:reveng:function:file', ('comp', {'fields': (('function', 'it:reveng:function'), ('file', 'file:bytes'))}), {
```

Comment:
we usually try to make some kinda "container" or "causal" direction to the comp ordering.  Since the "file contains the function" lets make it (file:bytes, it:reveng:function).

Reply:
:+1: